query/src/sessions/query_ctx_shared.rs

```diff
@@ -33,6 +31,8 @@ use common_planners::PlanNode;
 use common_users::RoleCacheMgr;
 use common_users::UserApiProvider;
 use futures::future::AbortHandle;
+use parking_lot::Mutex;
```

Comment:
As std's `Mutex` has been improved a lot, we still need `parking_lot`?

Reply:
We can use `std::sync::Mutex`, emmm, only one global replacement required

Reply:
Also: https://github.com/Amanieu/parking_lot/issues/338

Reply:
With fewer threads, `parking_lot` still performs better, and given our current instances, the replacement to `std::sync::Mutex` doesn't feel necessary.

but, perhaps on some crates it helps us to reduce dependencies.